cmd/helm/completion.go

```diff
@@ -53,24 +53,24 @@ func newCompletionCmd(out io.Writer) *cobra.Command {
 		Short: "Generate autocompletions script for the specified shell (bash or zsh)",
 		Long:  completionDesc,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return RunCompletion(out, cmd, args)
+			return runCompletion(out, cmd, args)
 		},
 		ValidArgs: shells,
 	}
 
 	return cmd
 }
 
-func RunCompletion(out io.Writer, cmd *cobra.Command, args []string) error {
+func runCompletion(out io.Writer, cmd *cobra.Command, args []string) error {
```

Comment:
Yeah -- this is good. We have no API compatibility requirements for anything in `cmd/helm`, and this should definitely be unexported.